app/controllers/api/v1/animals_controller.rb

```diff
@@ -1,24 +1,30 @@
 class Api::V1::AnimalsController < ApplicationController
+  
+  def show
+    animal = Animal.find(params[:id])
+    if animal
+      render json: AnimalSerializer.new(animal)
+    end
+  end
+
   def index
```

Comment:
Great call to change this to just a #show action.  This is a really clear and logical controller. refactor